airbrake.py

```diff
@@ -31,13 +36,34 @@ def retract_airbrakes(self):
 
     #TODO Confirm the launch detection magnitude
     def detect_launch(self):
-        if(self.accelerometer.accel_magnitude() > launch_detection_magnitude):
+        avg = 0.0
+        for i in range(2):
+            avg += self.accelerometer.accel_magnitude()
+        avg = avg / 3
+
+        if(avg > launch_detection_magnitude):
             return True
         else:
             return False
 
+    def get_accel_mag(self):
+        return self.accelerometer.accel_magnitude()
+
     def get_altitude(self):
-        return self.barometer.get_altitude()
+        print("AB tare value: " + str(self.airbrake_tare_value))
+        b = (self.barometer.get_altitude() - self.airbrake_tare_value)
+        self.loggerCSV.writeTo(b)
+        return b    
 
     def get_acceleration(self):
```

Comment:
Remove the function since we will not be using accelerometer outside of this file